packages/react-core/src/demos/PageLayout/examples/BannerDemo.md

```diff
@@ -0,0 +1,528 @@
+        <PageHeaderToolsGroup
+          visibility={{
+            default: 'hidden',
+            lg: 'visible'
+          }} /** the settings and help icon buttons are only visible on desktop sizes and replaced by a kebab dropdown for other sizes */
+        >
+          <PageHeaderToolsItem>
+            <Button aria-label="Settings actions" variant={ButtonVariant.plain}>
+              <CogIcon />
+            </Button>
+          </PageHeaderToolsItem>
+          <PageHeaderToolsItem>
+            <Button aria-label="Help actions" variant={ButtonVariant.plain}>
+              <HelpIcon />
+            </Button>
+          </PageHeaderToolsItem>
+        </PageHeaderToolsGroup>
+        <PageHeaderToolsGroup>
+          <PageHeaderToolsItem
+            visibility={{
+              lg: 'hidden'
+            }} /** this kebab dropdown replaces the icon buttons and is hidden for desktop sizes */
+          >
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={this.onKebabDropdownSelect}
+              toggle={<KebabToggle onToggle={this.onKebabDropdownToggle} />}
+              isOpen={isKebabDropdownOpen}
+              dropdownItems={kebabDropdownItems}
+            />
+          </PageHeaderToolsItem>
+          <PageHeaderToolsItem
+            visibility={{ default: 'hidden', md: 'visible' }} /** this user dropdown is hidden on mobile sizes */
+          >
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={this.onDropdownSelect}
+              isOpen={isDropdownOpen}
+              toggle={<DropdownToggle onToggle={this.onDropdownToggle}>John Smith</DropdownToggle>}
+              dropdownItems={userDropdownItems}
+            />
+          </PageHeaderToolsItem>
+        </PageHeaderToolsGroup>
+        <Avatar src={imgAvatar} alt="Avatar image" />
+      </PageHeaderTools>
+    );
+
+    const Header = (
+      <PageHeader logo={<Brand src={imgBrand} alt="Patternfly Logo" />} headerTools={headerTools} showNavToggle />
+    );
+    const Sidebar = <PageSidebar nav={PageNav} theme="dark" />;
+    const pageId = 'main-content-page-layout-default-nav';
+    const PageSkipToContent = <SkipToContent href={`#${pageId}`}>Skip to content</SkipToContent>;
+
+    return (
+      <React.Fragment>
+        <Page
+          header={Header}
+          sidebar={Sidebar}
+          isManagedSidebar
+          skipToContent={PageSkipToContent}
+          mainContainerId={pageId}
+        >
+          <Banner isSticky>
+            <Flex
+              justifyContent={{ default: 'justifyContentCenter', lg: 'justifyContentSpaceBetween' }}
+              flexWrap={{ default: 'nowrap' }}
+            >
+              <div className="pf-u-display-none pf-u-display-block-on-lg">Localhost</div>
+              <div className="pf-u-display-none pf-u-display-block-on-lg">
+                This message is sticky to the top or bottom of the page.
```

Comment:
I think it's just a little confusing that the two demos, which appear to have banners sticking to three different parts of the page all have the same message that says, "This message is sticky to the top or bottom of the page." Maybe the messages should each be a little more specific and different?

Reply:
That's how they are in the core demo. I can update them to reference their specific location.

Reply:
Agree, we should update that in the core demo, too. If you want to make that change in this demo, core can update to match.